### PR TITLE
cleanup and reformat 01 script bash syntax

### PR DESCRIPTION
**Pull request by tuminoid**
As part of dev-env bash cleanup patch series, cleanup 01 script.

### CODE REVIEW

**Comment by dtantsur** [comment]
Not sure what is WIP here, but looking good so far.

**Comment by tuminoid** [comment]
> Not sure what is WIP here, but looking good so far.

Nothing anymore, since it passed tests. PTAL.
/cc @dtantsur @lentzi90

File: 01_prepare_host.sh [hunk 1]
```diff
@@ -71,83 +75,88 @@ sudo "${ANSIBLE_VENV}/bin/pip" install --ignore-installed ansible=="${ANSIBLE_VE
...
-          sudo "${CONTAINER_RUNTIME}" pod rm "${pod}" -f
-      fi
-      sudo "${CONTAINER_RUNTIME}" pod create -n "${pod}"
-    done
-    ;;
-  *)
-    ;;
+    podman)
+        for pod in ironic-pod infra-pod; do
+            if  sudo "${CONTAINER_RUNTIME}" pod exists "${pod}" ; then
```
[the remark this answers is not in the document]
Comment (tuminoid) [reply 01_prepare_host.sh:145]: I'll get it in next PR.

**Review by smoshiur1237 — COMMENTED** [review]
/approve

**Pull request merged**